src/expCalc.cpp

```diff
@@ -0,0 +1,109 @@
+/*
+Expressions calculation��such as 5 + 12 / ��1+2��
+*/
+
+#include <iostream>
+#include <stack>
+#include <string> 
+#include <cstring>
+using namespace std;
+

```

Comment:
I want a function named `eval(const std::string& expression)` but not a class. Can you implement it ?